### PR TITLE
Prototype use of markdown for static pages

### PR DESCRIPTION
**Pull request by mloughran**
Thoughts on this before I bother converting any other pages?

I have renamed the key to `body_md` which makes the type explicit to content writers, but no magic happens based on the suffix (`t` has enough magic already imo).

### CODE REVIEW

**Comment by tatyree** [comment]
I'm dubious, @mloughran.  It increases our dependencies but I'm not sure it offers that much over the html.  Is markdown Leigh's preferred format?

**Comment by mloughran** [comment]
@tatyree I agree the value for the 404 is low, but for other pages I think it's a lot friendlier for content writers (and Leigh has expressed a slight preference for markdown). Besides that, it's harder to generate invalid HTML, and invalid markdown will raise errors and therefore be easily caught in tests.

**Comment by tatyree** [comment]
@mloughran Ok, I'm convinced.  @threedaymonk, what's your take?

File: spec/controllers/pages_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe HighVoltage::PagesController do
```
Comment (tatyree) [inline spec/controllers/pages_controller_spec.rb]: I'm OK with it if you want to disable rubocop on this one.  I can't find the exact rule name, though.

**Pull request closed without merging**